### PR TITLE
Improve readme for image_set_tag

### PR DESCRIPTION
**Pull request by tom-mullen**
Why:

* It reference carrierwave conventions, which weren't very transparent

This change addresses the need by:

* Replacing the carrierwave conventions with urls to placeholder imagery
* Adding an explanation on how to integrate lazy loading

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,13 +90,29 @@ Used to set `srcset` attribute on `img` tag.
...
+image_set_tag('http://placehold.it/1200x600',
+  srcset: {
+    'http://placehold.it/300x150' => '300w',
+    'http://placehold.it/600x300' => '600w',
+    'http://placehold.it/1200x600' => '1200w'
+  },
+  class: 'lazyload'
+)
+```
+N.B. You would need to enure the lazySizes script is installed for this to work
```
Comment (vfonic) [inline README.md]: I'd replace "installed" with "included in the project" or similar. We don't really install javascript scripts.

**Comment by vfonic** [comment]
Just what I need! :)

We could still mention how to use it with carrierwave as one of the options.

:ship:

**Comment by tom-mullen** [comment]
I don’t really think that’s relevant. If we did that, then we should also mention paperclip and any other image handler that can be found on rails. Seems a bit limiting.

**Comment by vfonic** [comment]
I think the correct question is: "Does this add a value?" For carrierwave users, it does. For others, it gives general idea how it could work in real word scenario end-to-end. There are demo apps showcasing usages and even though they don't include 100% of use cases, they show a good example. If someone who uses paperclip finds this, he can contribute and add examples for paperclip.

**Comment by tom-mullen** [comment]
Hmm, i’m not really convinced. Even if we did offer a carrierwave section, we are effectively telling people they just need to put the image path/url in here, like this.

You couldn’t make the snippet homogenous as everyone will have different sizes defined in their uploader, which then means we are basically trying to explain to people how to use carrierwave. On top of that, our use case might not even be correct, as they may prefer to use relative paths over complete urls.

So in response to your question, I would it probably adds more confusion than value.

File: README.md (hunk 1 above)
Comment (cheba) [inline README.md]: Also typo: enure

Comment (tom-mullen) [inline README.md]: I'm the only native speaker on here and I still miss the typo ;-)

**Pull request merged**